charts/prometheus/templates/server/deploy.yaml

```diff
@@ -38,6 +38,13 @@ spec:
 {{- end }}
 {{- if .Values.server.schedulerName }}
       schedulerName: "{{ .Values.server.schedulerName }}"
+{{- end }}
+{{- if semverCompare ">=1.13-0" .Capabilities.KubeVersion.GitVersion }}
+      {{- if or (.Values.server.enableServiceLinks) (eq (.Values.server.enableServiceLinks | toString) "<nil>") }}
+      enableServiceLinks: true
+      {{- else }}
+      enableServiceLinks: false
+      {{- end }}
```

Comment:
Could we move this version detection code to _helpers.tpl ?

Reply:
As stated in PR description, its just an alignment with STS that I can push as new PR if you prefer.

Reply:
Change push to a new PR: https://github.com/prometheus-community/helm-charts/pull/1125